atomic_reactor/plugins/pre_resolve_remote_source.py

```diff
@@ -76,84 +79,85 @@ def run(self):
             return
 
         if (not get_allow_multiple_remote_sources(self.workflow)
-                and self.workflow.source.config.remote_sources):
-            raise ValueError('Multiple remote sources are not supported, use single '
-                             'remote source in container.yaml')
+                and self.multiple_remote_sources_params):
+            raise ValueError('Multiple remote sources are not enabled, enable them or use '
```

Comment:
I'd remove 'enable them' as users can't do that on their own, but it is in reactor-config-map

Reply:
fixed